components/spi/oauth-service-deployment-patch.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "op": "add",
+    "path": "/spec/template/spec/containers/0/env",
+    "value": [
+      {
+        "name": "API_SERVER",
+        "value": "https://api-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com:443"
```

Comment:
what will happen when value will be empty? Will it use default kube API?

Reply:
yes